client/db/bolt/db.go

```diff
@@ -159,8 +159,10 @@ func (db *BoltDB) Run(ctx context.Context) {
 	db.Close()
 }
 
-// Store stores a value at the specified key in the general-use bucket.
-func (db *BoltDB) Store(k string, v []byte) error {
+// SetEncryptionParams stores encryption parameters
+// params (associated with key k) overwriting previous value
+// if any already exists for key k.
+func (db *BoltDB) SetEncryptionParams(k string, v []byte) error {
```

Comment:
Almost the solution. Making these fully specific to the encryption parameters would be to eliminate the key `k` input argument since that becomes a DB implementation detail.  The `k`-`v` only make sense for generic Get/Store, but that's what we wanted to eliminate.

Anyway, this is a moot point with https://github.com/decred/dcrdex/pull/1015

Thank you for your contribution here though!

Reply:
Yeah, I thought about that too. 

I think at this point it is no longer worth to develop this change any further, so I'll close in favor of https://github.com/decred/dcrdex/pull/1015.